server/services/weather.py

```diff
@@ -38,12 +38,18 @@ def acknowledge_recommendation(demoGuid, recommendationId):
 def trigger_simulation(demoGuid):
     """
     Trigger a simulation in the given demo
+    Creates a Snow Storm in the DC area
     """
 
     try:
         payload = dict()
         payload['demoGuid'] = demoGuid
-        response = call_openwhisk('simulate', payload)
+        event = dict()
```

Comment:
Changed to use the format of an event from the weather API. See sample_event.json